libkineto/src/CuptiActivityProfiler.cpp

```diff
@@ -840,6 +840,14 @@ void CuptiActivityProfiler::configure(
       timestamp = system_clock::now();
     }
 #ifdef HAS_CUPTI
+#ifdef _WIN32
+    CUPTI_CALL(
+        cuptiActivityRegisterTimestampCallback([]() -> uint64_t {
+          auto system = std::chrono::time_point_cast<std::chrono::nanoseconds>(
+              std::chrono::system_clock::now());
```

Comment:
Well we already convert the CUPTI time to system time on Windows https://github.com/pytorch/kineto/blob/main/libkineto/src/CuptiActivityPlatform.cpp#L21. The time needs to be in between the system clock start and end https://github.com/pytorch/kineto/blob/a1c3fea49769bbf31343554cdb569d5e88ac7242/libkineto/src/CuptiActivityProfiler.cpp#L408 so it will happen to be outside that window. So in this case it makes sense to either use system time on Windows too, or maybe we could either convert to monotonic on every platform or maybe change on windows to have captureWindowStartTime_  and captureWindowEndTime_ use monotonic time too.